### PR TITLE
Hn updates

### PR DESCRIPTION
**Pull request by hknahal**

### CODE REVIEW

**Comment by hknahal** [comment]
@kcullion @rosibaj I'm still adding more information to the documentation (have placeholder for table/diagram for clinical cross-file validations).

File: docs/submission/clinical-data-validation-rules.md [hunk 1]
```diff
@@ -6,8 +6,53 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+- Each follow up should be assigned a unique `submitter_follow_up_id`. Optionally, you can include the `submitter_treatment_id` the follow up may be linked to.
+
+### Time Intervals
+
+To prevent potential identification of donors, actual calendar dates are not permitted. The timing of different clinical events are collected in days counted from the date of primary diagnosis.
+
+### Donors Older than 90 years old
+
+Since the occurrence of individuals over the age of 90 is rare and therefore potentially make the individual identifiable, the `age_at_diagnosis` field is capped at 90.
```
Comment (rosibaj) [inline docs/submission/clinical-data-validation-rules.md:44]: sentence is a bit weird.  suggestion: 

Since the occurrence of individuals over the age of 90 is rare ` it is therefore considered a potentially identifiable value` and allowed value for the `age_at_diagnosis` field is capped at 90.

Comment (kcullion) [reply docs/submission/clinical-data-validation-rules.md:44]: Maybe reword like: 

Since the occurrence of individuals over the age of 90 is rare, it is therefore considered a potentially identifiable value. Thus, the allowed value for the age_at_diagnosis field is capped at 90.

File: docs/submission/clinical-data-validation-rules.md [hunk 2]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+The data dictionary contains certain data elements regarded as "identifiers". These fields have an ![ID](/assets/submission/dictionary-id.svg) descriptor in the data dictionary and include:
+
+- Donor: `submitter_donor_id`
+- Specimen: `submitter_specimen_id`
+- Sample: `submitter_sample_id` 
+- Primary Diagnosis: `submitter_primary_diagnosis_id`
+- Treatment: `submitter_treatment_id`
+- Follow Up: `submitter_follow_up_id`
+
+These fields must to be coded specifically for ICGC-ARGO purposes using the following rules:
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:22]: @rosibaj we haven't been using a dash ever, but I did see some demos where Christina did this too. Do we want to change it so we always use a dash? Or just accept both ways of writing ICGC ARGO?

Comment (rosibaj) [reply docs/submission/clinical-data-validation-rules.md:22]: lets be consistent i our writing.  lets go no dash as thats mostly everywhere.

its just natural for people to want to put a dash.

Comment (kcullion) [reply docs/submission/clinical-data-validation-rules.md:22]: yes I agree..especially since it's in the url. ok sounds good, Hardeep can you remove the dash please?

Comment (hknahal) [reply docs/submission/clinical-data-validation-rules.md:22]: Sure no problem, I'll remove the dash.

File: docs/submission/clinical-data-validation-rules.md [hunk 3]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+
+- These identifiers should not be derived from IDs such as biobank or hospital identifiers or any other personal identifying information. These IDs are to be coded in such a way that they cannot be tracked back to the individual donors, except by the submitting program. Only the program will keep the key that will permit the data to be linked back to the individual donors. This key must not be communicated to the data users.
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:24]: suggestion but please ignore if wrong..

These identifiers should not be derived from biobank or hospital identifiers or any other personal identifying information.

File: docs/submission/clinical-data-validation-rules.md [hunk 4]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+- Identifiers are assigned by each submitting program and must be unique within all the data submitted by that program (no duplicate IDs allowed).
+- These identifiers should be values that uniquely identify the entity they refer to.
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:26]: Is this saying the same thing as the point above?

File: docs/submission/clinical-data-validation-rules.md [hunk 5]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+- Identifiers referring to the same entity should be consistent across different submissions from the submitting program, and should not be re-used to refer to two different entities. For example, the same donor should not be assigned different identifiers in different files or subsequent data submissions.
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:27]: Suggestion:

Identifiers referring to the same entity should be consistent across separate program submissions and should not be re-used for different entities.

File: docs/submission/clinical-data-validation-rules.md [hunk 6]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+- Identifiers cannot begin with DO, SP, SA, PD, TR or FU. These prefixes are reserved for ICGC-ARGO IDs.
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:28]: change ICGC-ARGO to ICGC ARGO

File: docs/submission/clinical-data-validation-rules.md [hunk 7]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+
+### Primary Diagnosis, Treatment and Follow Up Identifiers:
+
+These identifier fields allow linking of the different clinical events and should be coded using the following rules:
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:32]: suggestion: 
These identifier fields allow for linking across the different clinical events and should be coded using the following rules:

File: docs/submission/clinical-data-validation-rules.md [hunk 8]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+
+- Each primary diagnosis should be assigned a unique `submitter_primary_diagnosis_id`. In the case of multiple primary diagnoses, each primary diagnosis should be a different `submitter_primary_diagnosis_id`. You will be required to submit the `submitter_primary_diagnosis_id` in the `Specimen` file - this provides information about which primary diagnosis the specimen is linked to. The `submitter_primary_diagnosis_id` is also required in the `Treatment` table, so it is understood which primary diagnosis the treatment is being administered for.
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:34]: Each primary diagnosis should be assigned a unique `submitter_primary_diagnosis_id`. In the case of multiple primary diagnoses, each primary diagnosis should be a different `submitter_primary_diagnosis_id`. 

These two sentences seem to say the same thing? could change to:
Each primary diagnosis should be assigned a unique `submitter_primary_diagnosis_id`, so in the case where a donor has multiple primary diagnoses, each primary diagnosis should have a different `submitter_primary_diagnosis_id`.

File: docs/submission/clinical-data-validation-rules.md [hunk 9]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+- Each treatment regimen in the `Treatment` table should be assigned a unique `submitter_treatment_id`. If the treatment regimen consists of chemotherapy, hormone therapy or radiation therapy, then you will use the same `submitter_treatment_id` in the appropriate `Chemotherapy`, `Radiation` or `Hormone Therapy` tables. For example, a treatment regimen consisting of Chemotherapy and Radiation therapy is assigned `cr01` as the `submitter_treatment_id` in the `Treatment` file. You would then submit the relevant clinical treatment information in the `Chemotherapy` and `Radiation` files using the same `submitter_treatment_id` (`cr01`) in the `Chemotherapy` and `Radiation` files. This allows the information in the two tables to be linked together so it is understood that the two therapies were combined.
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:35]: here should we also say Treatment "`file`" instead of table.

File: docs/submission/clinical-data-validation-rules.md [hunk 10]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+- Each follow up should be assigned a unique `submitter_follow_up_id`. Optionally, you can include the `submitter_treatment_id` the follow up may be linked to.
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:36]: suggestion: 
Each follow up should be assigned a unique `submitter_follow_up_id`. Optionally, if a follow up is linked to a specific treatment, you may include the `submitter_treatment_id` for that followup.

File: docs/submission/clinical-data-validation-rules.md [hunk 11]
```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
...
+
+### Time Intervals
+
+To prevent potential identification of donors, actual calendar dates are not permitted. The timing of different clinical events are collected in days counted from the date of primary diagnosis.
+
+### Donors Older than 90 years old
+
+Since the occurrence of individuals over the age of 90 is rare, it is therefore considered a potentially identifiable value and the allowed value. Thus, the allowed value for the `age_at_diagnosis` field is capped at 90.
```
Comment (kcullion) [inline docs/submission/clinical-data-validation-rules.md:44]: grammar fix:

Since the occurrence of individuals over the age of 90 is rare, it is therefore considered a potentially identifiable value.

File: docs/submission/dictionary-overview.md [hunk 12]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
 
 - Other fields must meet a regular expression for their value.
 
-## Dictionary Reference Databases
+## Dictionary Standards Used
+
+The dictionary codelist values were derived from external standards or common terminology used by ICGC-ARGO projects. These include:
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:57]: change ICGC-ARGO to ICGC ARGO

File: docs/submission/dictionary-overview.md [hunk 13]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+- [World Health Organization International Classification of Diseases, 10th Revision (ICD-10)](https://icd.who.int/browse10/2019/en)
+- [International Classification of Diseases for Oncology (ICD-O))](http://www.iacr.com.fr/index.php?option=com_content&view=category&layout=blog&id=100&Itemid=577)
+- [Cancer Data Standards Registry and Repository (caDSR)](https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html#/search)
+- [Cancer Care Ontario Data Book Reporting Standards](https://www.cancercareontario.ca/en/data-book-reporting-standards)
+- [RxNorm](https://www.nlm.nih.gov/research/umls/rxnorm/index.html)
+- [Common Terminology Criteria for Adverse Events (CTCAE)](https://ctep.cancer.gov/protocolDevelopment/electronic_applications/ctc.htm)
+- [ECOG-ACRIN Cancer Research Group](https://ecog-acrin.org/resources/ecog-performance-status)
+ 
+
+## Tumour staging classifications
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:69]: capitalize "Staging Classifications"

File: docs/submission/dictionary-overview.md [hunk 14]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+
+### Revised International staging system (RISS)
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:71]: capitalize "Staging System"

File: docs/submission/dictionary-overview.md [hunk 15]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+Note: Hodgkin lymphoma uses A or B designation with stage group.  A/B is no longer used in NHL
+
+| Suffix | Description |
+|--------|-------------|
+| A | Asymptomatic (No B symptoms) |
+| B | Any B symptom(s): <ul><li>Fevers. Unexplained fever with temperature above 38°C.</li><li>Night sweats. Drenching sweats (e.g., those that require change of bed clothes).</li><li>Weight loss. Unexplained weight loss of more than 10% of the usual body weight in the 6 months prior to diagnosis</li></ul> |
+Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition (p. 981). American College of Surgeons.
+
+
+### St. Jude Children's Research Hospital Staging system
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:101]: capitalize `System`

File: docs/submission/dictionary-overview.md [hunk 16]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+| Suffix | Description |
+|--------|-------------|
+| A | No symptoms. |
+| B | Fever (temperature >38.0ºC), drenching night sweats, unexplained loss of >10% of body weight within the preceding 6 months. |
+| E | Involvement of a single extranodal site that is contiguous or proximal to the known nodal site. |
+| S | Splenic involvement. |
+Reference: [https://www.ncbi.nlm.nih.gov/books/NBK65726/table/CDR0000062933__557/](https://www.ncbi.nlm.nih.gov/books/NBK65726/table/CDR0000062933__557/)
+
+
+### Rai staging system
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:130]: capitalize `Staging System`

File: docs/submission/dictionary-overview.md [hunk 17]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+|-------|-------------|
+| Stage 0 | Lymphocytosis (high blood count of lymphocytes) and no enlargement of the lymph nodes, spleen, or liver, and with near normal red blood cell and platelet counts. |
+| Stage I | Lymphocytosis plus enlarged lymph nodes. The spleen and liver are not enlarged and the red blood cell and platelet counts are normal or only slightly low. |
+| Stage II | Lymphocytosis plus an enlarged spleen (and possibly an enlarged liver), with or without enlarged lymph nodes. The red blood cell and platelet counts are normal or only slightly low |
+| Stage III | Lymphocytosis plus anemia (too few red blood cells), with or without enlarged lymph nodes, spleen, or liver. Platelet counts are near normal.|
+| Stage IV | Lymphocytosis plus thrombocytopenia (too few platelets), with or without anemia, enlarged lymph nodes, spleen, or liver. |
+Reference: [CLL Society, Inc. Rai Staging of CLL (chronic lymphocytic leukemia).](https://cllsociety.org/2016/03/rai-staging-cll-chronic-lymphocytic-leukemia/)
+
+
+### Durie-Salmon staging system
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:142]: capitalize `Staging System`

File: docs/submission/dictionary-overview.md [hunk 18]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+
+| Stage | Description |
+|-------|-------------|
+| Stage 1 | All of the following: <ul><li>Hemoglobin value > 10 g/dl</li><li>Serum calcium value normal or <10.5 mg/dL</li><li>Bone X-ray, normal bone structure (scale 0), or solitary bone plasmactyoma only</li><li>Low M-component production rates IgG value < 5g/dL; IgA value < 3 g/dL</li><li>Urine light chain M-component on electrophoresis < 4 g/24h</li></ul> |
+| Stage II | Fitting neither Stage I nor Stage III |
+| Stage III | One or more of the following: <ul><li>Hemoglobin value < 8.5 g/dL</li><li>Serum calcium value > 12 mg/dL</li><li>Advanced lytic bone lesions (scale 3)</li><li>High M-component production rates IgG value > 7 g/dL; IgA value > 5 g/dL</li><li>Urine light chain M-component > 12 g/24h</li></ul> |
+Reference: [https://www.myeloma.org/durie-salmon-staging](https://www.myeloma.org/durie-salmon-staging)
+
+
+### FIGO staging of Cervical carcinomas
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:152]: FIGO Staging of Cervical Carcinomas

File: docs/submission/dictionary-overview.md [hunk 19]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+| Stage IB2 | Clinical lesions greater than 4 cm in size.|
+| Stage IIA | No obvious parametrial involvement. Involvement of up to the upper two-thirds of the vagina.|
+| Stage IAB | Obvious parametrial involvement, but not into the pelvic sidewall.|
+| Stage IIIA | No extension into the pelvic sidewall but involvement of the lower third of the vagina.|
+| Stage IIIB | Extension into the pelvic sidewall or hydronephrosis or non-functioning kidney.|
+| Stage IVA | Spread of the tumour into adjacent pelvic organs.|
+| Stage IVB | Spread to distant organs.|
+Reference: [https://screening.iarc.fr/viaviliappendix1.php](https://screening.iarc.fr/viaviliappendix1.php)
+
+### Binet staging system
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:170]: capitalize `Staging System`

File: docs/submission/dictionary-overview.md [hunk 20]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+
+| Stage | Description |
+|-------|-------------|
+| Stage A | Fewer than 3 groups of enlarged lymph nodes (lymphadenopathy) and a high white blood cell count |
+| Stage B | More than 3 groups of enlarged lymph nodes and a high white blood cell count 
+| Stage C | Enlarged lymph nodes or spleen, a high white blood cell count, and low red blood cell or platelet counts |
+Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition (p. 975). American College of Surgeons.
+
+
+## Tumour grading classifications
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:180]: Tumour Grading Classifications

File: docs/submission/dictionary-overview.md [hunk 21]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+
+
+### Two-tier grading system
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:183]: Two-tier Grading System

File: docs/submission/dictionary-overview.md [hunk 22]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+
+| Grade |
+|-------|
+| Low grade |
+| High grade | 
+Reference: Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition
+
+
+### Three-grade system
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:192]: should this be "Three-tier Grading System"?

File: docs/submission/dictionary-overview.md [hunk 23]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+| Grade | Definition |
+|-------|------------|
+| GX | Cannot be assessed |
+| G1 | Well differentiated |
+| G2 | Moderately differentiated |
+| G3 | Poorly differentiated, undifferentiated |
+Reference: Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition
+
+
+### Four-tier grading system
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:202]: Four-tier Grading System

File: docs/submission/dictionary-overview.md [hunk 24]
```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
...
+### Grading for Gastrointestinal Stromal Tumours (GISTs)
+	
+| Mitotic Rate | Definition |
+|--------------|------------|
+| Low | 5 or fewer mitoses per 5 mm2|
+| High | Over 5 mitoses per 5 mm2 |
+Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition (p. 546). American College of Surgeons.
+
+
+### Grading system for GNETs
```
Comment (kcullion) [inline docs/submission/dictionary-overview.md:222]: for rest of headings, just to be consistent with how this page was up top,  capitalize the important words

File: docs/submission/faq.md [hunk 25]
```diff
@@ -5,3 +5,33 @@ platform_key: DOCS_FAQ
 ---
 
 ![Reminder Banner](/assets/submission/banner-reminder.svg)
+
+## Frequently Asked Questions
+
+1. How do I submit multiple values for a field? For example, I want to submit multiple symptoms for the `presenting_symptoms` field.
+
+   To include multiple values, separate values with a pipe delimiter '|' within your file. For example: `Anemia|Fatigue|Fever`.
```
Comment (kcullion) [inline docs/submission/faq.md:13]: Do we need to say something like "when permitted"? not all the fields allow for multiple values.  Like we can say, the fields that allow for multiple values will have these instructions specified in the notes section of the dictionary.

File: docs/submission/faq.md [hunk 26]
```diff
@@ -5,3 +5,33 @@ platform_key: DOCS_FAQ
...
+
+2. Is there any resource to refer to in order to follow staging guidelines?
```
Comment (kcullion) [inline docs/submission/faq.md:15]: suggestion: 

Is there any resource to refer to for staging guidelines?

File: docs/submission/faq.md [hunk 27]
```diff
@@ -5,3 +5,33 @@ platform_key: DOCS_FAQ
...
+
+    Please refer to the guidelines provides by the [AJCC Cancer Staging](https://cancerstaging.org/Pages/default.aspx) website. Other helpful resources include the [CAnswer Forum](http://cancerbulletin.facs.org/forums/).
```
Comment (kcullion) [inline docs/submission/faq.md:17]: typo: 
provides to `provided`

Also, change "Other helpful resources include the"... to "Another helpful resource is the..."

File: docs/submission/faq.md [hunk 28]
```diff
@@ -5,3 +5,33 @@ platform_key: DOCS_FAQ
...
+
+3. The `tumour_staging_system` field does not contain the staging system I use. Can we request the staging system be added?
+
+    Yes, if the tumour staging system or tumour grading system you use is not included in the controlled terminology for `tumour_staging_system` or `tumour_grading_system`, please [contact us](https://platform.icgc-argo.org/contact) to request it be added.
```
Comment (kcullion) [inline docs/submission/faq.md:21]: @rosibaj this contact link is a case where we could ask the devs to specify the "subject" in the url of the contact page and have this link fill out that "What do you need assistance with" subject for them? That way we ensure the query going to the right person right away. 

So the url would be something like https://platform.icgc-argo.org/contact?data_or_anlaysis_query. 

Do you want to do this?

File: docs/submission/faq.md [hunk 29]
```diff
@@ -5,3 +5,33 @@ platform_key: DOCS_FAQ
...
+
+4. What assembly version do you use?
+
+    ICGC-ARGO uses GrCh38.
```
Comment (kcullion) [inline docs/submission/faq.md:25]: - change ICGC-ARGO to ICGC ARGO.
- is it written like: GRCh38 Human Reference Genome (capitalized R)
https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.26/

File: docs/submission/faq.md (hunk 28 above)
Comment (rosibaj) [reply docs/submission/faq.md:21]: @kcullion i think we would rather not support that level of integration.... It will become quite unmanageable for maintenance over time it think to keep up with all the places we have added that.

if the jira workflow changes, then we will need to change all the spots we specify things like that.

Comment (kcullion) [reply docs/submission/faq.md:21]: ok sounds good

File: docs/submission/clinical-data-validation-rules.md (hunk 4 above)
Comment (hknahal) [reply docs/submission/clinical-data-validation-rules.md:26]: Yes, it sounds the same. I removed `These identifiers should be values that uniquely identify the entity they refer to.`

File: docs/submission/dictionary-overview.md (hunk 13 above)
Comment (hknahal) [reply docs/submission/dictionary-overview.md:69]: I fixed all the casings to make them consistent.

File: docs/submission/dictionary-overview.md (hunk 22 above)
Comment (hknahal) [reply docs/submission/dictionary-overview.md:192]: Yes it should. I corrected this.

File: docs/submission/faq.md (hunk 25 above)
Comment (hknahal) [reply docs/submission/faq.md:13]: Yes, I updated this sentence so it's clear now.
